.github/workflows/ci-code.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         AIIDA_WARN_v3: 1
       # Python 3.12 has a performance regression when running with code coverage
       # so run code coverage only for python 3.9.
-      run: pytest -v tests -m 'not nightly' ${{ matrix.python-version == '3.9' && '--cov aiida' || '' }}
```

Comment:
Running pytest in verbose mode produces more than 2000 lines of output, which is extremely annoying in Github UI.